src/components/home/BarcodeForm.js

```diff
@@ -0,0 +1,9 @@
+<FormGroup>
```

Comment:
Great! 

Based on the pair programming we did can you change this in to a fully fledged react component?

```jsx
// import the things you need

const BarcodeForm = props => {
  return (
    // the template code
  )
} 

export default BarcodeForm;
```